tst/testbugfix/00069.tst

```diff
@@ -0,0 +1,3 @@
+# 2005/05/05 (TB)
+gap> IsBound( ShowPackageVariables );
+true
```

Comment:
Another test which just checks whether some identifier is defined. What is the point here? Preventing us from accidentally removing any documented identifiers? But then, shouldn't we rather have a full test for this, e.g. get a list of all documented identifiers, and verify they are defined?